packages/ibm-products/src/components/Card/Card.tsx

```diff
@@ -262,7 +262,10 @@ export const Card = forwardRef(
             [`${blockClass}__clickable`]: clickable,
             [`${blockClass}__media-left`]: mediaPosition === 'left',
             [`${blockClass}--has-slug`]: !!slug,
-            [`${blockClass}--has-ai-label`]: !!aiLabel,
+            [`${blockClass}--has-decorator-non-ai-label`]:
+              !!decorator && decorator['type']?.displayName !== 'AILabel',
+            [`${blockClass}--has-decorator`]:
+              !!decorator && decorator['type']?.displayName === 'AILabel',
```

Comment:
Can we simplify this to just use `[`${blockClass}--has-decorator`]` and rely on the presence of `cds--slug`/`cds--ai-label` classes to add the gradients?